src/utils.js

```diff
@@ -18,6 +18,22 @@ export function isUnsafeCallee(node) {
   ];
 }
 
+export function isUnsafeConstEvalRequireImport(node) {
+  const isUnsafe = (node.declarations &&
+    node.declarations[0].init &&
+    node.declarations[0].init.callee &&
+    node.declarations[0].init.callee.callee &&
+    node.declarations[0].init.callee.callee.type === "Identifier" &&
+    node.declarations[0].init.callee.callee.name === "eval" &&
+    node.declarations[0].init.callee.arguments[0].type === "Literal" &&
+    node.declarations[0].init.callee.arguments[0].value === "require");
```

Comment:
You can use the following code to simplify

```js
const identifier = getCallExpressionIdentifier(node);
```

Then just need to assert that identifier is eval and doing check on `node.arguments.at(0)`

Reply:
And so I think we don't need to put in in utils

Reply:
I've tried to simplify it to maximum using the method you suggested me (thanks a lot), by keeping some (necessary?) conditional check to keep tests green, lmk if it seems correct to you